### PR TITLE
docs: update documentation for v1.1.0 release

### PR DESCRIPTION
**Pull request by AustinGreen**
**Motivation:**

The Llama framework is ready to be upgraded to `v1.1.0`. 

**Modifications:**

- Adds the `LlamaAccountWithDelegation` and `LlamaAccountTokenDelegationScript` contract addresses to the deployments section of the README
- Updates the language for the scripts section on the `actions.md` doc
- Creates a dedicated `scripts.md` doc that covers the new scripts that are part of `v1.1.0`
- Updates `accounts.md` to cover the `LlamaAccountWithDelegation` logic contract.
- Added January audit.

**Result:**

We'll be ready to release `v1.1.0` of llama.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,12 +67,14 @@ The comments in that file explain what each variable is for and when they're nee
...
+| LlamaAccountTokenDelegationScript     | [0xC60Ab709CA5BbD73fC2b46D232344740A8903b51](https://etherscan.io/address/0xC60Ab709CA5BbD73fC2b46D232344740A8903b51) | [0xC60Ab709CA5BbD73fC2b46D232344740A8903b51](https://optimistic.etherscan.io/address/0xC60Ab709CA5BbD73fC2b46D232344740A8903b51) | [0xC60Ab709CA5BbD73fC2b46D232344740A8903b51](https://arbiscan.io/address/0xC60Ab709CA5BbD73fC2b46D232344740A8903b51) | [0xC60Ab709CA5BbD73fC2b46D232344740A8903b51](https://basescan.org/address/0xC60Ab709CA5BbD73fC2b46D232344740A8903b51) | [0xC60Ab709CA5BbD73fC2b46D232344740A8903b51](https://polygonscan.com/address/0xC60Ab709CA5BbD73fC2b46D232344740A8903b51) |
```
Comment (0xrajath) [inline README.md:77]: I'm assuming you've precomputed these addresses since they're not deployed yet right?

File: README.md [hunk 2]
```diff
@@ -113,11 +117,12 @@ $ forge doc -o reference/ -b -s
 
 ### Audits
 
-We received two audits from Spearbit and one from Code4rena. You can find links to the reports below:
+We've received three audits from Spearbit and one from Code4rena. You can find links to the reports below:
 
 - [Llama Spearbit Audit (June 2023)](https://github.com/llamaxyz/llama/blob/main/audits/Llama-Spearbit-Audit.pdf)
 - [Llama Code4rena Audit](https://github.com/llamaxyz/llama/blob/main/audits/Llama-Code4rena-Audit.md)
 - [Llama Spearbit Audit (August 2023)](https://github.com/llamaxyz/llama/blob/main/audits/Llama-Spearbit-Audit-2.pdf)
+- Llama v1.1.0 Spearbit Audit (report coming soon)
```
Comment (0xrajath) [inline README.md:125]: Will we be linking the Spearbit audit after cutting the v1.1.0 release?

**Review by 0xrajath — APPROVED** [review]
LGTM. Left 2 queries.

File: README.md (hunk 1 above)
Comment (AustinGreen) [reply README.md:77]: Yes

File: README.md (hunk 2 above)
Comment (AustinGreen) [reply README.md:125]: Yeah the report might take a week so didn't want to block this PR

**Review by 0xrajath — APPROVED** [review]

**Pull request merged**